charts/graphprotocol-node/templates/configmap.yaml

```diff
@@ -13,7 +13,7 @@ data:
     [store]
     [store.primary]
       connection = "postgresql://{{ $pgUser }}:{{ $pgPass }}@{{ $pgHost }}/{{ $pgDB }}"
-      weight = 0
+      weight = 1
```

Comment:
I would add a link somewhere to https://github.com/graphprotocol/graph-node/blob/5613dca92213810b3cca57979f5facc2d670e34a/docs/config.md#configuring-multiple-databases